### DISCUSSION TITLE
Level Strip to show sub-Xsheet drawings' thumbnails (sub-Xsheet name ID)

### DISCUSSION THREAD

**Discussion by gab3d** [opentoonz/opentoonz#6831] [Ideas]
It would be quite useful for the Level Strip to be able to show sub-Xsheet drawings' icons/thumbnails.
In this way, Level Strip could be used to select and expose in the main Xsheet the desired sub-Xsheet drawing, as if it was a regular level. I'm thinking of a library of poses stored in a sub-Xsheet, for example...

I am aware that currently sub-Xsheets don't have a name ID exposed to the user (and that, since in Level Strip levels are selected by name, a name should be assigned to each sub-Xsheet in the scene), but I also think this ID should be quite useful as a way to access sub-Xsheets from expressions. And that, in turn, could improve OT's rigging capabilities a lot.
So it seems like a single, seemingly humble feature that could nevertheless impact positively in several aspects of OT.

**Comment by DarrenTAnims** [comment]
This would be a really useful feature. I guess what it would need is to update a series of thumbnails as you save the sub-xsheet (or when you import the sub-xsheet into another scene). Then you could show those thumbnails in the level strip.
